### ISSUE TITLE
JS: Introduce JavaScript's usage in the Command-Line

### ISSUE THREAD

**Issue by paramsiddharth** [girlscript/winter-of-contributing#72]
# Introduce JavaScript's usage in the Command-Line
Discuss the growing use of JavaScript in the command-line,
for front-end web development using frameworks, back-end
web development using servers and API libraries, and desktop and/or
mobile application development.

## Directory
```
Topics/4.Command_Line
```

## Description
- Talk about standalone JavaScript scripts that run in the command-line.
- Can we use Node.js and NPM for front-end development?
- Can we use Bootstrap, et cetera, via NPM?
- What frameworks can be used with Node.js to ease front-end development?
- What popular back-end API frameworks do we have in Node.js?
- Discuss the domination of Electron as the most popular hybrid desktop application
  development framework of the times.
- Discuss React Native and Ionic as hybrid/near-native mobile application development
  frameworks in JavaScript.

## Hints
- Check out the about page of [Node.js](https://nodejs.org/en/about/).

## Formats
- [x] Document
- [ ] Video
- [ ] Audio

**Comment by Asmit2952** [comment]
Hey I want to work on this!!

**Comment by Rijudas149** [comment]
@paramsiddharth sir I want to work on this

**Comment by TishaJhabak1014** [comment]
Sir, if possible assign these to me too.

**Comment by TishaJhabak1014** [comment]
Sir, I will go for the documentation for this.  Thus, specified. Thank you for the assignment.

**Comment by 7sakshi7** [comment]
I am a GWOC'21 contributor from Batch 3 of JavaScript. Please assign this issue to me. I assure you that this issue's title will be the same as my PR's title as well as I will be Contributing under the Guidelines and Code of Conduct.
Discord : agarwalSakshi | Contributor#9567

**Comment by Divyanshigarg** [comment]
I am a Gwoc contributor from batch 16 which include front-end with JavaScript.Kindly provide me this domain for contribution.

**Comment by paramsiddharth** [comment]
> I am a Gwoc contributor from batch 16 which include front-end with JavaScript.Kindly provide me this domain for contribution.

@Divyanshigarg Please make sure you are allowed to contribute to the JavaScript domain. There's often a clash between us and FEWD. Cheers!

**Comment by TishaJhabak1014** [comment]
@paramsiddharth Sir, raised a PR for this issue.

**Comment by TishaJhabak1014** [comment]
@paramsiddharth Sir plz close the issue, since it has already been merged, Thank you

**Comment by TishaJhabak1014** [comment]
@paramsiddharth Sir, please it is a request to do the needful to close the issue. Thank you.

**Comment by paramsiddharth** [comment]
@TishaJhabak1014 No, I won't close the issue yet. As the workflow for us has been, this same issue will be reopened and reused for the other formats.

**Comment by TishaJhabak1014** [comment]
@paramsiddharth sir, okay I got it. Actually, I guess, this issue is counted as open and even I am not working on 3 open issues, the bot prompts 3 issue at a time is exhausted for me. Thus, I cannot work on rest domains.

**Comment by paramsiddharth** [comment]
> @paramsiddharth sir, okay I got it. Actually, I guess, this issue is counted as open and even I am not working on 3 open issues, the bot prompts 3 issue at a time is exhausted for me. Thus, I cannot work on rest domains.

In such a situation, you can just unassign yourself from an issue that you are done working on. :)

**Comment by TishaJhabak1014** [comment]
Thank you sir : )
